README.md

```diff
@@ -31,18 +31,19 @@ Check out the [getting started guide](./docs/getting_started.md) for launching a
-| CAPV v1beta1 (v1.7) |              ☓              |              ☓              |             ☓              |             ✓              |             ✓             |
+|                                  | Cluster API v1alpha3 (v0.7) | Cluster API v1alpha4 (v0.8) | Cluster API v1beta1 (v1.0) | Cluster API v1beta1 (v1.3) | Cluster API v1beta1 (v1.4) | Cluster API v1beta1 (v1.5) |
+|----------------------------------|:---------------------------:|:---------------------------:|:--------------------------:|:--------------------------:|:--------------------------:|:--------------------------:|
+| CAPV v1alpha3 (v0.7)(deprecated) |              ✓              |              ✓              |             ✓              |             ✓              |             ☓              |             ☓              |
+| CAPV v1alpha4 (v0.8)(deprecated) |              ☓              |              ✓              |             ✓              |             ✓              |             ☓              |             ☓              |
+| CAPV v1beta1 (v1.0)              |              ☓              |              ☓              |             ✓              |             ✓              |             ☓              |             ☓              |
+| CAPV v1beta1 (v1.1)              |              ☓              |              ☓              |             ☓              |             ✓              |             ☓              |             ☓              |
+| CAPV v1beta1 (v1.2)              |              ☓              |              ☓              |             ☓              |             ✓              |             ☓              |             ☓              |
+| CAPV v1beta1 (v1.3)              |              ☓              |              ☓              |             ☓              |             ✓              |             ☓              |             ☓              |
+| CAPV v1beta1 (v1.4)              |              ☓              |              ☓              |             ☓              |             ✓              |             ☓              |             ☓              |
+| CAPV v1beta1 (v1.5)              |              ☓              |              ☓              |             ☓              |             ✓              |             ✓              |             ☓              |
+| CAPV v1beta1 (v1.6)              |              ☓              |              ☓              |             ☓              |             ✓              |             ✓              |             ☓              |
+| CAPV v1beta1 (v1.7)              |              ☓              |              ☓              |             ☓              |             ✓              |             ✓              |             ☓              |
+| CAPV v1beta1 (v1.8)              |              ☓              |              ☓              |             ☓              |             ☓              |             ☓              |             ✓              |
```

Comment:
I don't know how this was filled out before, but I would stick with supporting what we test which is the combination CAPV v1.8 & CAPI v1.5

We should probably rethink the format of this table as it doesn't scale well, but let's keep it as is for now